src/errors.ts

```diff
@@ -32,12 +35,7 @@ export class NamedError extends Error {
   }
 
   public get fullStack(): string | undefined {
-    let stack = this.stack;
-    const causedStack = this.cause?.fullStack ?? this.cause?.stack;
-    if (causedStack) {
-      stack = `${stack ? stack + '\n' : ''}Caused by: ${causedStack}`;
-    }
-    return stack;
+    return format(this);
```

Comment:
sfError took this approach, rather than trying to recursively recreate the causes' stacks

Reply:
I think you meant `util.inspect`:
https://nodejs.org/dist/latest-v20.x/docs/api/errors.html#errorcause
https://nodejs.org/dist/latest-v20.x/docs/api/util.html#utilinspectobject-options

sfCommandError:
https://github.com/salesforcecli/sf-plugins-core/blob/d764f60d7d8d5b45304a5d436e03293127152dae/src/SfCommandError.ts#L90